tutorials/writing-tutorials.md

```diff
@@ -34,7 +34,14 @@ Example tutorials are available at [NMA github course-content/tutorials](https:/
 
 For each tutorial:
 1. Import all necessary libraries, plotting settings and plotting functions in the top code cell(s) of the tutorial
-   - write `# @title Imports` (or other relevant title) at the top of each cell, and click `...`>`Form`>`Hide code` to hide the code cell (special colab trickery)
+   - write `# @title Imports` (or other relevant title) at the top of each cell. We received feedback that students want to see the library imports, so don't hide this cell. For the rest (figure settings, helper functions, data loading), hide the cell: Left-click `...`>`Form`>`Hide code` (special colab trickery).
```

Comment:
As a general rule, I don't think we should use `@title` where we don't intend to hide: it squashes the code in a way that both looks bad and is hard to read.

Reply:
Fixed.